### ISSUE TITLE
Like hashtags DEBUG - <Response [200]> but not doing it

### ISSUE THREAD

**Issue by fabriziospadaro** [ohld/igbot#1219]
Hello after providing my credentials correctly I get:
INFO - Logged-in successfully as 'xxxxxxx'

Then it gathers the hashtags and starts liking them and i get:
DEBUG - <Response [200]>

If I check from Instagram the post that I liked recently there is no new content being liked at all.

91%|█████████ | 78/86 [14:10<01:26, 10.84s/it]2020-01-10 17:11:04,664 - DEBUG - <Response [200]>
2020-01-10 17:11:04,972 - DEBUG - <Response [200]>
2020-01-10 17:11:05,279 - DEBUG - <Response [200]>
 92%|█████████▏| 79/86 [14:21<01:16, 10.89s/it]2020-01-10 17:11:15,622 - DEBUG - <Response [200]>
2020-01-10 17:11:15,929 - DEBUG - <Response [200]>
2020-01-10 17:11:16,237 - DEBUG - <Response [200]>
 93%|█████████▎| 80/86 [14:32<01:05, 10.91s/it]2020-01-10 17:11:26,511 - DEBUG - <Response [200]>
2020-01-10 17:11:26,767 - DEBUG - <Response [200]>
2020-01-10 17:11:27,002 - DEBUG - <Response [200]>
 94%|█████████▍| 81/86 [14:43<00:54, 10.86s/it]2020-01-10 17:11:37,331 - DEBUG - <Response [200]>
2020-01-10 17:11:37,640 - DEBUG - <Response [200]>
2020-01-10 17:11:37,945 - DEBUG - <Response [200]>
 95%|█████████▌| 82/86 [14:54<00:43, 10.89s/it]2020-01-10 17:11:48,494 - DEBUG - <Response [200]>
2020-01-10 17:11:48,905 - DEBUG - <Response [200]>
2020-01-10 17:11:49,311 - DEBUG - <Response [200]>
 97%|█████████▋| 83/86 [15:05<00:33, 11.03s/it]2020-01-10 17:11:59,610 - DEBUG - <Response [200]>
2020-01-10 17:11:59,963 - DEBUG - <Response [200]>
2020-01-10 17:12:00,269 - DEBUG - <Response [200]>
 98%|█████████▊| 84/86 [15:16<00:22, 11.01s/it]2020-01-10 17:12:10,612 - DEBUG - <Response [200]>
2020-01-10 17:12:11,022 - DEBUG - <Response [200]>
2020-01-10 17:12:11,328 - DEBUG - <Response [200]>
 99%|█████████▉| 85/86 [15:27<00:11, 11.02s/it]2020-01-10 17:12:21,672 - DEBUG - <Response [200]>
2020-01-10 17:12:21,961 - DEBUG - <Response [200]>
2020-01-10 17:12:22,288 - DEBUG - <Response [200]>
100%|██████████| 86/86 [15:38<00:00, 10.91s/it]
2020-01-10 17:12:32,332 - INFO - DONE: Total liked 0 medias.
2020-01-10 17:12:32,335 - INFO - Total requests: 416

**Comment by fighterii** [comment]
Actually it says exactly that in your log:

 `INFO - DONE: Total liked 0 medias.`
The 200 request is probably a get request for the media before it should like medias (post requests). Actually we could improve debug logging for requests here, but I'm wondering why you don't see info/warning/error about why nothing is liked... Strange

**Comment by fighterii** [comment]
PR #1223 will provide better debug logging. As soon as it is merged you should try again.

**Comment by very-sad-cat** [comment]
I'm actually getting this issue too. It never likes the photos 

![image](https://user-images.githubusercontent.com/31162981/72250788-642d1a00-3637-11ea-9cc1-dc1ec60500d3.png)

**Comment by bruvv** [comment]
There is a new update, please run that and report back what the message is. Thanks

**Comment by zzeesstt** [comment]
up +

**Comment by bruvv** [comment]
What do you mean? Did you run the newer version and did you get the exact same error ?

**Comment by very-sad-cat** [comment]
nope still the same exact error

**Comment by bruvv** [comment]
Can you confirm that you have downloaded the new version and post the script in correct Markup you're using and including the error ?

**Comment by Abdulsamipy** [comment]
same problem .. i have updated the bot too

**Comment by Abdulsamipy** [comment]
same problem .. i have updated the bot too
ok so this error is only in hashtag option ..everything else is working fine

**Comment by fighterii** [comment]
Actually there are still no logs from the new version. The new version logs which rest route gives the 200 response code. We need this to investigate further, please provide at least 15 log lines from your execution with the newest bot version (and don't say "same error")

**Comment by Abdulsamipy** [comment]
2020-01-16 20:11:46,294 - INFO - Going to like media with hashtag #pewdiepie.
Getting hashtag media.: 0it [00:00, ?it/s]2020-01-16 20:11:50,708 - DEBUG - <Response [200]>
2020-01-16 20:11:51,421 - DEBUG - <Response [200]>
2020-01-16 20:11:51,427 - INFO - Going to like 79 medias.
  0%|                                                                                           | 0/79 [00:00<?, ?it/s]2020-01-16 20:11:54,260 - DEBUG - <Response [200]>
2020-01-16 20:11:54,706 - DEBUG - <Response [200]>
2020-01-16 20:11:55,208 - DEBUG - <Response [200]>
  1%|█                                                                                  | 1/79 [00:13<17:55, 13.78s/it]2020-01-16 20:12:30,577 - DEBUG - <Response [200]>
2020-01-16 20:12:31,108 - DEBUG - <Response [200]>
2020-01-16 20:12:31,457 - DEBUG - <Response [200]>
  3%|██                                                                                 | 2/79 [00:50<26:20, 20.52s/it]2020-01-16 20:13:29,623 - DEBUG - <Response [200]>
2020-01-16 20:13:30,543 - DEBUG - <Response [200]>
2020-01-16 20:13:30,993 - DEBUG - <Response [200]>
  4%|███▏                                                                               | 3/79 [01:49<40:49, 32.23s/it]2020-01-16 20:14:39,289 - DEBUG - <Response [200]>
2020-01-16 20:14:40,737 - DEBUG - <Response [200]>
2020-01-16 20:14:41,200 - DEBUG - <Response [200]>
  5%|████▏                                                                              | 4/79 [02:59<54:31, 43.62s/it]2020-01-16 20:15:46,018 - DEBUG - <Response [200]>
2020-01-16 20:15:46,532 - DEBUG - <Response [200]>
2020-01-16 20:15:47,243 - DEBUG - <Response [200]>
  6%|█████▏                                                                           | 5/79 [04:05<1:02:05, 50.35s/it]2020-01-16 20:16:21,548 - DEBUG - <Response [200]>
2020-01-16 20:16:21,964 - DEBUG - <Response [200]>
2020-01-16 20:16:22,330 - DEBUG - <Response [200]>
  8%|██████▎                                                                            | 6/79 [04:40<55:41, 45.77s/it]2020-01-16 20:16:50,812 - DEBUG - <Response [200]>
2020-01-16 20:16:51,762 - DEBUG - <Response [200]>
2020-01-16 20:16:52,265 - DEBUG - <Response [200]>
  9%|███████▎                                                                           | 7/79 [05:10<49:13, 41.02s/it]2020-01-16 20:18:00,290 - DEBUG - <Response [200]>
2020-01-16 20:18:03,437 - DEBUG - <Response [200]>
2020-01-16 20:18:03,946 - DEBUG - <Response [200]>

Blacklist hashtag found in media, skipping!
 10%|████████▍                                                                          | 8/79 [06:22<59:25, 50.22s/it]2020-01-16 20:18:28,732 - DEBUG - <Response [200]>
2020-01-16 20:18:29,182 - DEBUG - <Response [200]>
2020-01-16 20:18:29,953 - DEBUG - <Response [200]>
 11%|█████████▍                                                                         | 9/79 [06:48<50:06, 42.95s/it]2020-01-16 20:19:22,565 - DEBUG - <Response [200]>
2020-01-16 20:19:22,947 - DEBUG - <Response [200]>
2020-01-16 20:19:23,295 - DEBUG - <Response [200]>
 13%|██████████▍                                                                       | 10/79 [07:41<52:58, 46.07s/it]2020-01-16 20:19:59,036 - DEBUG - <Response [200]>
2020-01-16 20:19:59,411 - DEBUG - <Response [200]>
2020-01-16 20:19:59,879 - DEBUG - <Response [200]>
 14%|███████████▍                                                                      | 11/79 [08:18<48:59, 43.23s/it]2020-01-16 20:21:02,119 - DEBUG - <Response [200]>
2020-01-16 20:21:02,526 - DEBUG - <Response [200]>
2020-01-16 20:21:02,876 - DEBUG - <Response [200]>
 15%|████████████▍                                                                     | 12/79 [09:21<54:53, 49.15s/it]2020-01-16 20:22:09,738 - DEBUG - <Response [200]>
2020-01-16 20:22:10,320 - DEBUG - <Response [200]>
2020-01-16 20:22:10,659 - DEBUG - <Response [200]>
 16%|█████████████▏                                                                  | 13/79 [10:29<1:00:13, 54.74s/it]2020-01-16 20:23:19,915 - DEBUG - <Response [200]>
2020-01-16 20:23:20,478 - DEBUG - <Response [200]>
2020-01-16 20:23:20,806 - DEBUG - <Response [200]>
 18%|██████████████▏                                                                 | 14/79 [11:39<1:04:18, 59.37s/it]2020-01-16 20:24:08,841 - DEBUG - <Response [200]>
2020-01-16 20:24:09,247 - DEBUG - <Response [200]>
2020-01-16 20:24:09,591 - DEBUG - <Response [200]>
 19%|███████████████▌                                                                  | 15/79 [12:28<59:56, 56.19s/it]2020-01-16 20:25:19,132 - DEBUG - <Response [200]>
2020-01-16 20:25:19,678 - DEBUG - <Response [200]>
2020-01-16 20:25:20,022 - DEBUG - <Response [200]>
 20%|████████████████▏                                                               | 16/79 [13:38<1:03:29, 60.46s/it]2020-01-16 20:26:18,798 - DEBUG - <Response [200]>
2020-01-16 20:26:19,158 - DEBUG - <Response [200]>
2020-01-16 20:26:19,486 - DEBUG - <Response [200]>
 22%|█████████████████▏                                                              | 17/79 [14:38<1:02:10, 60.16s/it]2020-01-16 20:27:24,072 - DEBUG - <Response [200]>
2020-01-16 20:27:24,447 - DEBUG - <Response [200]>
2020-01-16 20:27:24,791 - DEBUG - <Response [200]>
 23%|██████████████████▏                                                             | 18/79 [15:43<1:02:44, 61.71s/it]2020-01-16 20:27:53,542 - DEBUG - <Response [200]>
2020-01-16 20:27:53,917 - DEBUG - <Response [200]>
2020-01-16 20:27:54,261 - DEBUG - <Response [200]>
 24%|███████████████████▋                                                              | 19/79 [16:12<52:02, 52.04s/it]2020-01-16 20:29:00,289 - DEBUG - <Response [200]>
2020-01-16 20:29:00,833 - DEBUG - <Response [200]>
2020-01-16 20:29:01,193 - DEBUG - <Response [200]>
 25%|████████████████████▊                                                             | 20/79 [17:19<55:33, 56.50s/it]2020-01-16 20:29:34,848 - DEBUG - <Response [200]>
2020-01-16 20:29:35,231 - DEBUG - <Response [200]>
2020-01-16 20:29:35,591 - DEBUG - <Response [200]>
 27%|█████████████████████▊                                                            | 21/79 [17:54<48:12, 49.87s/it]2020-01-16 20:30:41,181 - DEBUG - <Response [200]>
2020-01-16 20:30:41,762 - DEBUG - <Response [200]>
2020-01-16 20:30:42,105 - DEBUG - <Response [200]>
 28%|██████████████████████▊                                                           | 22/79 [19:00<52:07, 54.86s/it]

**Comment by Abdulsamipy** [comment]
The .txt log file has no info in it .. So i copy pasted it from cmd
`
2020-01-16 20:11:46,294 - INFO - Going to like media with hashtag #pewdiepie.
Getting hashtag media.: 0it [00:00, ?it/s]2020-01-16 20:11:50,708 - DEBUG - <Response [200]>
2020-01-16 20:11:51,421 - DEBUG - <Response [200]>
2020-01-16 20:11:51,427 - INFO - Going to like 79 medias.
  0%|                                                                                           | 0/79 [00:00<?, ?it/s]2020-01-16 20:11:54,260 - DEBUG - <Response [200]>
2020-01-16 20:11:54,706 - DEBUG - <Response [200]>
2020-01-16 20:11:55,208 - DEBUG - <Response [200]>
  1%|█                                                                                  | 1/79 [00:13<17:55, 13.78s/it]2020-01-16 20:12:30,577 - DEBUG - <Response [200]>
2020-01-16 20:12:31,108 - DEBUG - <Response [200]>
2020-01-16 20:12:31,457 - DEBUG - <Response [200]>
  3%|██                                                                                 | 2/79 [00:50<26:20, 20.52s/it]2020-01-16 20:13:29,623 - DEBUG - <Response [200]>
2020-01-16 20:13:30,543 - DEBUG - <Response [200]>
2020-01-16 20:13:30,993 - DEBUG - <Response [200]>
  4%|███▏                                                                               | 3/79 [01:49<40:49, 32.23s/it]2020-01-16 20:14:39,289 - DEBUG - <Response [200]>
2020-01-16 20:14:40,737 - DEBUG - <Response [200]>
2020-01-16 20:14:41,200 - DEBUG - <Response [200]>
  5%|████▏                                                                              | 4/79 [02:59<54:31, 43.62s/it]2020-01-16 20:15:46,018 - DEBUG - <Response [200]>
2020-01-16 20:15:46,532 - DEBUG - <Response [200]>
2020-01-16 20:15:47,243 - DEBUG - <Response [200]>
  6%|█████▏                                                                           | 5/79 [04:05<1:02:05, 50.35s/it]2020-01-16 20:16:21,548 - DEBUG - <Response [200]>
2020-01-16 20:16:21,964 - DEBUG - <Response [200]>
2020-01-16 20:16:22,330 - DEBUG - <Response [200]>
  8%|██████▎                                                                            | 6/79 [04:40<55:41, 45.77s/it]2020-01-16 20:16:50,812 - DEBUG - <Response [200]>
2020-01-16 20:16:51,762 - DEBUG - <Response [200]>
2020-01-16 20:16:52,265 - DEBUG - <Response [200]>
  9%|███████▎                                                                           | 7/79 [05:10<49:13, 41.02s/it]2020-01-16 20:18:00,290 - DEBUG - <Response [200]>
2020-01-16 20:18:03,437 - DEBUG - <Response [200]>
2020-01-16 20:18:03,946 - DEBUG - <Response [200]>

Blacklist hashtag found in media, skipping!
 10%|████████▍                                                                          | 8/79 [06:22<59:25, 50.22s/it]2020-01-16 20:18:28,732 - DEBUG - <Response [200]>
2020-01-16 20:18:29,182 - DEBUG - <Response [200]>
2020-01-16 20:18:29,953 - DEBUG - <Response [200]>
 11%|█████████▍                                                                         | 9/79 [06:48<50:06, 42.95s/it]2020-01-16 20:19:22,565 - DEBUG - <Response [200]>
2020-01-16 20:19:22,947 - DEBUG - <Response [200]>
2020-01-16 20:19:23,295 - DEBUG - <Response [200]>
 13%|██████████▍                                                                       | 10/79 [07:41<52:58, 46.07s/it]2020-01-16 20:19:59,036 - DEBUG - <Response [200]>
2020-01-16 20:19:59,411 - DEBUG - <Response [200]>
2020-01-16 20:19:59,879 - DEBUG - <Response [200]>
 14%|███████████▍                                                                      | 11/79 [08:18<48:59, 43.23s/it]2020-01-16 20:21:02,119 - DEBUG - <Response [200]>
2020-01-16 20:21:02,526 - DEBUG - <Response [200]>
2020-01-16 20:21:02,876 - DEBUG - <Response [200]>
 15%|████████████▍                                                                     | 12/79 [09:21<54:53, 49.15s/it]2020-01-16 20:22:09,738 - DEBUG - <Response [200]>
2020-01-16 20:22:10,320 - DEBUG - <Response [200]>
2020-01-16 20:22:10,659 - DEBUG - <Response [200]>
 16%|█████████████▏                                                                  | 13/79 [10:29<1:00:13, 54.74s/it]2020-01-16 20:23:19,915 - DEBUG - <Response [200]>
2020-01-16 20:23:20,478 - DEBUG - <Response [200]>
2020-01-16 20:23:20,806 - DEBUG - <Response [200]>
 18%|██████████████▏                                                                 | 14/79 [11:39<1:04:18, 59.37s/it]2020-01-16 20:24:08,841 - DEBUG - <Response [200]>
2020-01-16 20:24:09,247 - DEBUG - <Response [200]>
2020-01-16 20:24:09,591 - DEBUG - <Response [200]>
 19%|███████████████▌                                                                  | 15/79 [12:28<59:56, 56.19s/it]2020-01-16 20:25:19,132 - DEBUG - <Response [200]>
2020-01-16 20:25:19,678 - DEBUG - <Response [200]>
2020-01-16 20:25:20,022 - DEBUG - <Response [200]>
 20%|████████████████▏                                                               | 16/79 [13:38<1:03:29, 60.46s/it]2020-01-16 20:26:18,798 - DEBUG - <Response [200]>
2020-01-16 20:26:19,158 - DEBUG - <Response [200]>
2020-01-16 20:26:19,486 - DEBUG - <Response [200]>
 22%|█████████████████▏                                                              | 17/79 [14:38<1:02:10, 60.16s/it]2020-01-16 20:27:24,072 - DEBUG - <Response [200]>
2020-01-16 20:27:24,447 - DEBUG - <Response [200]>
2020-01-16 20:27:24,791 - DEBUG - <Response [200]>
 23%|██████████████████▏                                                             | 18/79 [15:43<1:02:44, 61.71s/it]2020-01-16 20:27:53,542 - DEBUG - <Response [200]>
2020-01-16 20:27:53,917 - DEBUG - <Response [200]>
2020-01-16 20:27:54,261 - DEBUG - <Response [200]>
 24%|███████████████████▋                                                              | 19/79 [16:12<52:02, 52.04s/it]2020-01-16 20:29:00,289 - DEBUG - <Response [200]>
2020-01-16 20:29:00,833 - DEBUG - <Response [200]>
2020-01-16 20:29:01,193 - DEBUG - <Response [200]>
 25%|████████████████████▊                                                             | 20/79 [17:19<55:33, 56.50s/it]2020-01-16 20:29:34,848 - DEBUG - <Response [200]>
2020-01-16 20:29:35,231 - DEBUG - <Response [200]>
2020-01-16 20:29:35,591 - DEBUG - <Response [200]>
 27%|█████████████████████▊                                                            | 21/79 [17:54<48:12, 49.87s/it]2020-01-16 20:30:41,181 - DEBUG - <Response [200]>
2020-01-16 20:30:41,762 - DEBUG - <Response [200]>
2020-01-16 20:30:42,105 - DEBUG - <Response [200]>
 28%|██████████████████████▊                                                           | 22/79 [19:00<52:07, 54.86s/it]
```

**Comment by fighterii** [comment]
What is the output from
`pip3 show instabot`
Or
`pip show instabot`
? (More specifically: what is the version?)

**Comment by Abdulsamipy** [comment]
Name: instabot
Version: 0.104.0
Summary: Instagram bot scripts for promotion and API python wrapper.
Home-page: https://github.com/instagrambot/instabot
Author: Daniil Okhlopkov, Evgeny Kemerov
Author-email: danokhlopkov@gmail.com, eskemerov@gmail.com
License: Apache Software License 2.0
Location: c:\users\..\...\local\programs\python\python38\lib\site-packages
Requires: tqdm, requests, requests-toolbelt, schedule, pysocks, responses, future, six, huepy, pytz
Required-by:

**Comment by fighterii** [comment]
Super strange why there is still the old output even if you have the latest version and the extended logging is part of that code...

Edit: did you check out the github repo by yourself to use the examples? Examples/like_hashtags.py modifies sys.path to import the bot from this location where the example is run (not the pip installed location!) If so - is this folder up-to-date to the current master branch?

**Comment by fighterii** [comment]
@Abdulsamipy please see my edited post

**Comment by Abdulsamipy** [comment]
i have reinstalled the instabot-mater.zip(just to make sure everything is up-to-date)  and tried to run the bot (Example/multi_script_CLI.py)... ..still the same problem

**Comment by jhumms** [comment]
I'm having the same issue and I reinstalled it as well. I tried it on both the Like_Example.py and the Ultimate.py example scripts, the like from hashtag isn't working in either.

**Comment by fighterii** [comment]
Yeah, i believe you that it doesn't work. But for analysis it is crucial that we have the extended logging to figure out what exactly does not work (or does not return the infos we need)... 
Unfortunately I can't try it out by myself at the moment, I have to rely on someone providing logging to make at least a guess what's not working exactly.

**Comment by Abdulsamipy** [comment]
i tried to reset the setting ..and it's showing the following message 
`      1. Setting bot parameter
        2. Add user accounts
        3. Add competitor database
        4. Add hashtag database
        5. Add Comment database
        6. Add blacklist
        7. Add whitelist
        8. Clear all database
        9. Main menu

What setting do you need?
1
2020-01-17 20:30:49,420 - INFO - error, read exception bellow
2020-01-17 20:30:49,421 - INFO - expected str, bytes or os.PathLike object, not list
`

**Comment by fighterii** [comment]
That is a totally different error. One issue focuses on one (possible) bug. If you encounter an other problem please open a new issue for this!

**Comment by Abdulsamipy** [comment]
**objective** like by hashtag
**bot** Example/multi_script_CLI.py
**log File**
[instabot_2817407609104.log](https://github.com/instagrambot/instabot/files/4077552/instabot_2817407609104.log)

**Cmd**
Insert hashtags separated by spaces
Example: cat dog
what hashtags?
pewdiepie
2020-01-17 20:34:53,265 - INFO - Going to like media with hashtag #pewdiepie.
Getting hashtag media.: 0it [00:00, ?it/s]2020-01-17 20:34:57,660 - DEBUG - <Response [200]>
2020-01-17 20:34:58,142 - DEBUG - <Response [200]>
2020-01-17 20:34:58,146 - INFO - Going to like 73 medias.
  0%|                                                                                           | 0/73 [00:00<?, ?it/s]2020-01-17 20:34:58,894 - DEBUG - <Response [200]>
2020-01-17 20:34:59,365 - DEBUG - <Response [200]>
2020-01-17 20:35:00,106 - DEBUG - <Response [200]>
  1%|█▏                                                                                 | 1/73 [00:11<14:21, 11.96s/it]2020-01-17 20:35:10,549 - DEBUG - <Response [200]>
2020-01-17 20:35:10,942 - DEBUG - <Response [200]>
2020-01-17 20:35:11,759 - DEBUG - <Response [200]>
  3%|██▎                                                                                | 2/73 [00:23<14:02, 11.87s/it]2020-01-17 20:35:22,401 - DEBUG - <Response [200]>
2020-01-17 20:35:23,120 - DEBUG - <Response [200]>
2020-01-17 20:35:24,653 - DEBUG - <Response [200]>
  4%|███▍                                                                               | 3/73 [00:36<14:12, 12.17s/it]2020-01-17 20:35:35,147 - DEBUG - <Response [200]>
2020-01-17 20:35:35,638 - DEBUG - <Response [200]>
2020-01-17 20:35:36,579 - DEBUG - <Response [200]>
  5%|████▌                                                                              | 4/73 [00:48<13:54, 12.10s/it]2020-01-17 20:35:47,085 - DEBUG - <Response [200]>
2020-01-17 20:35:47,694 - DEBUG - <Response [200]>
2020-01-17 20:35:48,415 - DEBUG - <Response [200]>
  7%|█████▋                                                                             | 5/73 [01:00<13:37, 12.02s/it]2020-01-17 20:35:58,900 - DEBUG - <Response [200]>
2020-01-17 20:35:59,419 - DEBUG - <Response [200]>
2020-01-17 20:36:00,249 - DEBUG - <Response [200]>
  8%|██████▊                                                                            | 6/73 [01:12<13:21, 11.96s/it]2020-01-17 20:36:10,755 - DEBUG - <Response [200]>
2020-01-17 20:36:11,378 - DEBUG - <Response [200]>
2020-01-17 20:36:11,805 - DEBUG - <Response [200]>
 10%|███████▉                                                                           | 7/73 [01:23<13:01, 11.84s/it]2020-01-17 20:36:22,342 - DEBUG - <Response [200]>
2020-01-17 20:36:24,458 - DEBUG - <Response [200]>
2020-01-17 20:36:24,874 - DEBUG - <Response [200]>
 11%|█████████                                                                          | 8/73 [01:36<13:13, 12.21s/it]2020-01-17 20:36:35,316 - DEBUG - <Response [200]>
2020-01-17 20:36:36,135 - DEBUG - <Response [200]>
2020-01-17 20:36:36,639 - DEBUG - <Response [200]>
 12%|██████████▏                                                                        | 9/73 [01:48<12:52, 12.08s/it]2020-01-17 20:36:47,717 - DEBUG - <Response [200]>
2020-01-17 20:36:48,956 - DEBUG - <Response [200]>
2020-01-17 20:36:49,509 - DEBUG - <Response [200]>
 14%|███████████▏                                                                      | 10/73 [02:01<12:55, 12.32s/it]2020-01-17 20:36:59,891 - DEBUG - <Response [200]>
2020-01-17 20:37:00,302 - DEBUG - <Response [200]>
2020-01-17 20:37:00,730 - DEBUG - <Response [200]>
 15%|████████████▎                                                                     | 11/73 [02:12<12:23, 11.99s/it]2020-01-17 20:37:11,156 - DEBUG - <Response [200]>
2020-01-17 20:37:11,586 - DEBUG - <Response [200]>
2020-01-17 20:37:12,071 - DEBUG - <Response [200]>
 16%|█████████████▍                                                                    | 12/73 [02:23<11:59, 11.79s/it]2020-01-17 20:37:22,528 - DEBUG - <Response [200]>
2020-01-17 20:37:23,133 - DEBUG - <Response [200]>
2020-01-17 20:37:23,648 - DEBUG - <Response [200]>
 18%|██████████████▌                                                                   | 13/73 [02:35<11:43, 11.73s/it]2020-01-17 20:37:34,153 - DEBUG - <Response [200]>
2020-01-17 20:37:34,810 - DEBUG - <Response [200]>
2020-01-17 20:37:35,218 - DEBUG - <Response [200]>
 19%|███████████████▋                                                                  | 14/73 [02:47<11:29, 11.68s/it]2020-01-17 20:37:45,680 - DEBUG - <Response [200]>
2020-01-17 20:37:46,386 - DEBUG - <Response [200]>
2020-01-17 20:37:46,892 - DEBUG - <Response [200]>
 21%|████████████████▊                                                                 | 15/73 [02:58<11:17, 11.68s/it]2020-01-17 20:37:57,856 - DEBUG - <Response [200]>
2020-01-17 20:37:58,267 - DEBUG - <Response [200]>
2020-01-17 20:37:58,867 - DEBUG - <Response [200]>
 22%|█████████████████▉                                                                | 16/73 [03:10<11:10, 11.77s/it]2020-01-17 20:38:09,355 - DEBUG - <Response [200]>
2020-01-17 20:38:09,981 - DEBUG - <Response [200]>
2020-01-17 20:38:10,541 - DEBUG - <Response [200]>
 23%|███████████████████                                                               | 17/73 [03:22<10:57, 11.74s/it]2020-01-17 20:38:21,089 - DEBUG - <Response [200]>
2020-01-17 20:38:21,613 - DEBUG - <Response [200]>
2020-01-17 20:38:22,118 - DEBUG - <Response [200]>
 25%|████████████████████▏                                                             | 18/73 [03:33<10:43, 11.69s/it]2020-01-17 20:38:32,598 - DEBUG - <Response [200]>
2020-01-17 20:38:33,176 - DEBUG - <Response [200]>
2020-01-17 20:38:33,681 - DEBUG - <Response [200]>
 26%|█████████████████████▎                                                            | 19/73 [03:45<10:29, 11.65s/it]2020-01-17 20:38:44,439 - DEBUG - <Response [200]>
2020-01-17 20:38:44,993 - DEBUG - <Response [200]>
2020-01-17 20:38:45,515 - DEBUG - <Response [200]>
 27%|██████████████████████▍                                                           | 20/73 [03:57<10:20, 11.71s/it]2020-01-17 20:38:55,937 - DEBUG - <Response [200]>
2020-01-17 20:38:56,421 - DEBUG - <Response [200]>
2020-01-17 20:38:56,836 - DEBUG - <Response [200]>
 29%|███████████████████████▌                                                          | 21/73 [04:08<10:02, 11.59s/it]2020-01-17 20:39:07,306 - DEBUG - <Response [200]>
2020-01-17 20:39:07,691 - DEBUG - <Response [200]>
2020-01-17 20:39:08,140 - DEBUG - <Response [200]>
 30%|████████████████████████▋                                                         | 22/73 [04:20<09:46, 11.50s/it]2020-01-17 20:39:18,502 - DEBUG - <Response [200]>
2020-01-17 20:39:18,885 - DEBUG - <Response [200]>
2020-01-17 20:39:19,215 - DEBUG - <Response [200]>
 32%|█████████████████████████▊                                                        | 23/73 [04:31<09:28, 11.38s/it]2020-01-17 20:39:29,646 - DEBUG - <Response [200]>
2020-01-17 20:39:30,047 - DEBUG - <Response [200]>
2020-01-17 20:39:30,412 - DEBUG - <Response [200]>
 33%|██████████████████████████▉                                                       | 24/73 [04:42<09:14, 11.32s/it]2020-01-17 20:39:41,032 - DEBUG - <Response [200]>
2020-01-17 20:39:41,515 - DEBUG - <Response [200]>
2020-01-17 20:39:43,033 - DEBUG - <Response [200]>
 34%|████████████████████████████                                                      | 25/73 [04:54<09:22, 11.71s/it]2020-01-17 20:39:53,765 - DEBUG - <Response [200]>
2020-01-17 20:39:54,277 - DEBUG - <Response [200]>
2020-01-17 20:39:54,702 - DEBUG - <Response [200]>
 36%|█████████████████████████████▏                                                    | 26/73 [05:06<09:09, 11.70s/it]2020-01-17 20:40:05,443 - DEBUG - <Response [200]>
2020-01-17 20:40:06,484 - DEBUG - <Response [200]>
2020-01-17 20:40:06,911 - DEBUG - <Response [200]>
 37%|██████████████████████████████▎                                                   | 27/73 [05:18<09:05, 11.85s/it]2020-01-17 20:40:17,622 - DEBUG - <Response [200]>
2020-01-17 20:40:18,136 - DEBUG - <Response [200]>
2020-01-17 20:40:18,572 - DEBUG - <Response [200]>
 38%|███████████████████████████████▍                                                  | 28/73 [05:30<08:50, 11.79s/it]2020-01-17 20:40:29,401 - DEBUG - <Response [200]>
2020-01-17 20:40:29,804 - DEBUG - <Response [200]>
2020-01-17 20:40:30,229 - DEBUG - <Response [200]>
 40%|████████████████████████████████▌                                                 | 29/73 [05:42<08:37, 11.75s/it]2020-01-17 20:40:40,765 - DEBUG - <Response [200]>
2020-01-17 20:40:41,203 - DEBUG - <Response [200]>
2020-01-17 20:40:41,788 - DEBUG - <Response [200]>
 41%|█████████████████████████████████▋                                                | 30/73 [05:53<08:22, 11.69s/it]2020-01-17 20:40:52,267 - DEBUG - <Response [200]>
2020-01-17 20:40:52,761 - DEBUG - <Response [200]>
2020-01-17 20:40:53,186 - DEBUG - <Response [200]>
 42%|██████████████████████████████████▊                                               | 31/73 [06:05<08:07, 11.60s/it]2020-01-17 20:41:03,641 - DEBUG - <Response [200]>
2020-01-17 20:41:04,007 - DEBUG - <Response [200]>
2020-01-17 20:41:04,423 - DEBUG - <Response [200]>
 44%|███████████████████████████████████▉                                              | 32/73 [06:16<07:51, 11.49s/it]2020-01-17 20:41:15,186 - DEBUG - <Response [200]>
2020-01-17 20:41:15,902 - DEBUG - <Response [200]>
2020-01-17 20:41:16,339 - DEBUG - <Response [200]>
 45%|█████████████████████████████████████                                             | 33/73 [06:28<07:44, 11.62s/it]2020-01-17 20:41:26,884 - DEBUG - <Response [200]>
2020-01-17 20:41:27,358 - DEBUG - <Response [200]>
2020-01-17 20:41:27,797 - DEBUG - <Response [200]>
 47%|██████████████████████████████████████▏                                           | 34/73 [06:39<07:31, 11.57s/it]2020-01-17 20:41:38,348 - DEBUG - <Response [200]>
2020-01-17 20:41:40,017 - DEBUG - <Response [200]>
2020-01-17 20:41:40,505 - DEBUG - <Response [200]>
 48%|███████████████████████████████████████▎                                          | 35/73 [06:52<07:32, 11.91s/it]2020-01-17 20:41:51,132 - DEBUG - <Response [200]>
2020-01-17 20:41:51,632 - DEBUG - <Response [200]>
2020-01-17 20:41:52,243 - DEBUG - <Response [200]>
 49%|████████████████████████████████████████▍                                         | 36/73 [07:04<07:18, 11.86s/it]2020-01-17 20:42:03,096 - DEBUG - <Response [200]>
2020-01-17 20:42:03,506 - DEBUG - <Response [200]>
2020-01-17 20:42:03,916 - DEBUG - <Response [200]>
 51%|█████████████████████████████████████████▌                                        | 37/73 [07:15<07:04, 11.80s/it]2020-01-17 20:42:14,468 - DEBUG - <Response [200]>
2020-01-17 20:42:14,983 - DEBUG - <Response [200]>
2020-01-17 20:42:15,409 - DEBUG - <Response [200]>
 52%|██████████████████████████████████████████▋                                       | 38/73 [07:27<06:49, 11.71s/it]2020-01-17 20:42:25,928 - DEBUG - <Response [200]>
2020-01-17 20:42:26,346 - DEBUG - <Response [200]>
2020-01-17 20:42:27,198 - DEBUG - <Response [200]>
 53%|███████████████████████████████████████████▊                                      | 39/73 [07:39<06:38, 11.73s/it]2020-01-17 20:42:37,643 - DEBUG - <Response [200]>
2020-01-17 20:42:38,126 - DEBUG - <Response [200]>
2020-01-17 20:42:38,550 - DEBUG - <Response [200]>
 55%|████████████████████████████████████████████▉                                     | 40/73 [07:50<06:23, 11.62s/it]2020-01-17 20:42:49,018 - DEBUG - <Response [200]>
2020-01-17 20:42:50,304 - DEBUG - <Response [200]>
2020-01-17 20:42:50,753 - DEBUG - <Response [200]>
 56%|██████████████████████████████████████████████                                    | 41/73 [08:02<06:17, 11.80s/it]2020-01-17 20:43:01,504 - DEBUG - <Response [200]>
2020-01-17 20:43:01,876 - DEBUG - <Response [200]>
2020-01-17 20:43:02,626 - DEBUG - <Response [200]>
 58%|███████████████████████████████████████████████▏                                  | 42/73 [08:14<06:06, 11.82s/it]2020-01-17 20:43:13,387 - DEBUG - <Response [200]>
2020-01-17 20:43:13,880 - DEBUG - <Response [200]>
2020-01-17 20:43:14,613 - DEBUG - <Response [200]>
 59%|████████████████████████████████████████████████▎                                 | 43/73 [08:26<05:56, 11.87s/it]2020-01-17 20:43:25,021 - DEBUG - <Response [200]>
2020-01-17 20:43:25,442 - DEBUG - <Response [200]>
2020-01-17 20:43:26,032 - DEBUG - <Response [200]>
 60%|█████████████████████████████████████████████████▍                                | 44/73 [08:37<05:40, 11.74s/it]2020-01-17 20:43:36,696 - DEBUG - <Response [200]>
2020-01-17 20:43:37,406 - DEBUG - <Response [200]>
2020-01-17 20:43:37,917 - DEBUG - <Response [200]>
 62%|██████████████████████████████████████████████████▌                               | 45/73 [08:49<05:29, 11.78s/it]2020-01-17 20:43:48,385 - DEBUG - <Response [200]>
2020-01-17 20:43:49,384 - DEBUG - <Response [200]>
2020-01-17 20:43:50,103 - DEBUG - <Response [200]>
 63%|███████████████████████████████████████████████████▋                              | 46/73 [09:01<05:21, 11.90s/it]2020-01-17 20:44:00,960 - DEBUG - <Response [200]>
2020-01-17 20:44:01,570 - DEBUG - <Response [200]>
2020-01-17 20:44:02,096 - DEBUG - <Response [200]>
 64%|████████████████████████████████████████████████████▊                             | 47/73 [09:13<05:10, 11.93s/it]2020-01-17 20:44:13,042 - DEBUG - <Response [200]>
2020-01-17 20:44:13,552 - DEBUG - <Response [200]>
2020-01-17 20:44:13,962 - DEBUG - <Response [200]>
 66%|█████████████████████████████████████████████████████▉                            | 48/73 [09:25<04:57, 11.91s/it]2020-01-17 20:44:24,706 - DEBUG - <Response [200]>
2020-01-17 20:44:25,234 - DEBUG - <Response [200]>
2020-01-17 20:44:25,639 - DEBUG - <Response [200]>
2020-01-17 20:44:26,449 - DEBUG - <Response [200]>

`bot.following` is empty, will download.
2020-01-17 20:44:29,445 - DEBUG - <Response [200]>
                                                                                                                                                                       2020-01-17 20:44:31,152 - DEBUG - <Response [200]>                                                                                              | 0/542 [00:00<?, ?it/s]
Getting followings of 27194590479: 100%|████████████████████████████████████████████████████████████████████████████████████████████| 542/542 [00:01<00:00, 318.67it/s]
2020-01-17 20:44:31,683 - DEBUG - <Response [200]>                                                                                     | 1/542 [00:01<15:20,  1.70s/it]

USER_NAME: team._.ksi, FOLLOWER: 14422, FOLLOWING: 308

follower_count > bot.max_followers_to_follow, skipping!
Adding '6952322820' to `skipped.txt`.
 67%|███████████████████████████████████████████████████████                           | 49/73 [09:43<05:28, 13.68s/it]2020-01-17 20:44:42,328 - DEBUG - <Response [200]>
2020-01-17 20:44:43,139 - DEBUG - <Response [200]>
2020-01-17 20:44:43,665 - DEBUG - <Response [200]>
 68%|████████████████████████████████████████████████████████▏                         | 50/73 [09:55<05:02, 13.14s/it]2020-01-17 20:44:54,313 - DEBUG - <Response [200]>
2020-01-17 20:44:54,863 - DEBUG - <Response [200]>
2020-01-17 20:44:55,369 - DEBUG - <Response [200]>
 70%|█████████████████████████████████████████████████████████▎                        | 51/73 [10:07<04:39, 12.71s/it]2020-01-17 20:45:06,387 - DEBUG - <Response [200]>
2020-01-17 20:45:06,951 - DEBUG - <Response [200]>
2020-01-17 20:45:07,351 - DEBUG - <Response [200]>
 71%|██████████████████████████████████████████████████████████▍                       | 52/73 [10:19<04:22, 12.50s/it]2020-01-17 20:45:17,784 - DEBUG - <Response [200]>
2020-01-17 20:45:18,146 - DEBUG - <Response [200]>
2020-01-17 20:45:19,300 - DEBUG - <Response [200]>
 73%|███████████████████████████████████████████████████████████▌                      | 53/73 [10:31<04:06, 12.33s/it]2020-01-17 20:45:30,155 - DEBUG - <Response [200]>
2020-01-17 20:45:30,968 - DEBUG - <Response [200]>
2020-01-17 20:45:32,309 - DEBUG - <Response [200]>
 74%|████████████████████████████████████████████████████████████▋                     | 54/73 [10:44<03:58, 12.54s/it]2020-01-17 20:45:42,745 - DEBUG - <Response [200]>
2020-01-17 20:45:43,253 - DEBUG - <Response [200]>
2020-01-17 20:45:43,665 - DEBUG - <Response [200]>
 75%|█████████████████████████████████████████████████████████████▊                    | 55/73 [10:55<03:39, 12.18s/it]2020-01-17 20:45:54,114 - DEBUG - <Response [200]>
2020-01-17 20:45:54,549 - DEBUG - <Response [200]>
2020-01-17 20:45:55,230 - DEBUG - <Response [200]>
 77%|██████████████████████████████████████████████████████████████▉                   | 56/73 [11:07<03:23, 11.99s/it]2020-01-17 20:46:05,986 - DEBUG - <Response [200]>
2020-01-17 20:46:06,514 - DEBUG - <Response [200]>
2020-01-17 20:46:07,007 - DEBUG - <Response [200]>
 78%|████████████████████████████████████████████████████████████████                  | 57/73 [11:18<03:10, 11.93s/it]2020-01-17 20:46:17,761 - DEBUG - <Response [200]>
2020-01-17 20:46:18,180 - DEBUG - <Response [200]>
2020-01-17 20:46:18,904 - DEBUG - <Response [200]>
 79%|█████████████████████████████████████████████████████████████████▏                | 58/73 [11:30<02:58, 11.92s/it]2020-01-17 20:46:31,074 - DEBUG - <Response [200]>
2020-01-17 20:46:31,789 - DEBUG - <Response [200]>
2020-01-17 20:46:32,640 - DEBUG - <Response [200]>
 81%|██████████████████████████████████████████████████████████████████▎               | 59/73 [11:44<02:54, 12.47s/it]2020-01-17 20:46:43,055 - DEBUG - <Response [200]>
2020-01-17 20:46:44,135 - DEBUG - <Response [200]>
2020-01-17 20:46:44,797 - DEBUG - <Response [200]>
2020-01-17 20:46:45,228 - DEBUG - <Response [200]>
2020-01-17 20:46:48,316 - DEBUG - <Response [200]>

USER_NAME: be_like_ambani, FOLLOWER: 21392, FOLLOWING: 8

follower_count > bot.max_followers_to_follow, skipping!
Adding '11925409218' to `skipped.txt`.
 82%|███████████████████████████████████████████████████████████████████▍              | 60/73 [12:00<02:54, 13.44s/it]2020-01-17 20:46:59,128 - DEBUG - <Response [200]>
2020-01-17 20:46:59,847 - DEBUG - <Response [200]>
2020-01-17 20:47:00,257 - DEBUG - <Response [200]>
 84%|████████████████████████████████████████████████████████████████████▌             | 61/73 [12:12<02:35, 12.98s/it]2020-01-17 20:47:10,811 - DEBUG - <Response [200]>
2020-01-17 20:47:11,521 - DEBUG - <Response [200]>
2020-01-17 20:47:11,947 - DEBUG - <Response [200]>
 85%|█████████████████████████████████████████████████████████████████████▋            | 62/73 [12:23<02:18, 12.59s/it]2020-01-17 20:47:22,385 - DEBUG - <Response [200]>
2020-01-17 20:47:22,828 - DEBUG - <Response [200]>
2020-01-17 20:47:23,587 - DEBUG - <Response [200]>
 86%|██████████████████████████████████████████████████████████████████████▊           | 63/73 [12:35<02:03, 12.31s/it]2020-01-17 20:47:34,146 - DEBUG - <Response [200]>
2020-01-17 20:47:34,769 - DEBUG - <Response [200]>
2020-01-17 20:47:35,319 - DEBUG - <Response [200]>
 88%|███████████████████████████████████████████████████████████████████████▉          | 64/73 [12:47<01:49, 12.13s/it]2020-01-17 20:47:45,825 - DEBUG - <Response [200]>
2020-01-17 20:47:46,536 - DEBUG - <Response [200]>
2020-01-17 20:47:47,152 - DEBUG - <Response [200]>
 89%|█████████████████████████████████████████████████████████████████████████         | 65/73 [12:59<01:36, 12.04s/it]2020-01-17 20:47:57,708 - DEBUG - <Response [200]>
2020-01-17 20:47:58,148 - DEBUG - <Response [200]>
2020-01-17 20:47:58,623 - DEBUG - <Response [200]>
 90%|██████████████████████████████████████████████████████████████████████████▏       | 66/73 [13:10<01:23, 11.87s/it]2020-01-17 20:48:09,275 - DEBUG - <Response [200]>
2020-01-17 20:48:10,329 - DEBUG - <Response [200]>
2020-01-17 20:48:10,838 - DEBUG - <Response [200]>
 92%|███████████████████████████████████████████████████████████████████████████▎      | 67/73 [13:22<01:11, 11.98s/it]2020-01-17 20:48:28,733 - DEBUG - <Response [200]>
2020-01-17 20:48:29,473 - DEBUG - <Response [200]>
2020-01-17 20:48:30,579 - DEBUG - <Response [200]>
 93%|████████████████████████████████████████████████████████████████████████████▍     | 68/73 [13:42<01:11, 14.30s/it]2020-01-17 20:48:41,590 - DEBUG - <Response [200]>
2020-01-17 20:48:42,081 - DEBUG - <Response [200]>
2020-01-17 20:48:42,755 - DEBUG - <Response [200]>
 95%|█████████████████████████████████████████████████████████████████████████████▌    | 69/73 [13:54<00:54, 13.67s/it]2020-01-17 20:48:53,820 - DEBUG - <Response [200]>
2020-01-17 20:48:55,088 - DEBUG - <Response [200]>
2020-01-17 20:48:55,835 - DEBUG - <Response [200]>
 96%|██████████████████████████████████████████████████████████████████████████████▋   | 70/73 [14:07<00:40, 13.49s/it]2020-01-17 20:49:06,420 - DEBUG - <Response [200]>
2020-01-17 20:49:07,650 - DEBUG - <Response [200]>
2020-01-17 20:49:08,596 - DEBUG - <Response [200]>
 97%|███████████████████████████████████████████████████████████████████████████████▊  | 71/73 [14:20<00:26, 13.27s/it]2020-01-17 20:49:19,417 - DEBUG - <Response [200]>
2020-01-17 20:49:19,943 - DEBUG - <Response [200]>
2020-01-17 20:49:20,369 - DEBUG - <Response [200]>
 99%|████████████████████████████████████████████████████████████████████████████████▉ | 72/73 [14:32<00:12, 12.82s/it]2020-01-17 20:49:30,950 - DEBUG - <Response [200]>
2020-01-17 20:49:31,702 - DEBUG - <Response [200]>
2020-01-17 20:49:32,142 - DEBUG - <Response [200]>
100%|██████████████████████████████████████████████████████████████████████████████████| 73/73 [14:43<00:00, 12.11s/it]
2020-01-17 20:49:42,147 - INFO - DONE: Total liked 0 medias.

**Comment by Abdulsamipy** [comment]
is this enough info?

**Comment by fighterii** [comment]
It's not the amount of failing media likes (1 or 2 will be enough), it's the content of your debug output! This does not contain the newest information. Instead of

`2020-01-17 20:49:30,950 - DEBUG - <Response [200]>`

There has to be something like

`2020-01-17 20:49:30,950 - DEBUG - GET Request to endpoint: /example returned: <Response [200]>`

**Comment by Abdulsamipy** [comment]
so i looked into all log files can't seem to find anything like 

`2020-01-17 20:49:30,950 - DEBUG - GET Request to endpoint: /example returned: <Response [200]>`

**Comment by Abdulsamipy** [comment]
so i looked into all log files and can't seem to find anything like 

`DEBUG - GET Request to endpoint: /example returned: <Response [200]>`

**Comment by jhumms** [comment]
Here's from my log, I let it run for a few hours.

2020-01-16 19:48:16,783 - INFO - LOGIN FLOW! Just logged-in: False
2020-01-16 19:48:21,701 - INFO - Logged-in successfully as '******************!
2020-01-16 19:48:21,702 - INFO - ULTIMATE script. Safe to run 24/7!
2020-01-16 20:48:21,879 - INFO - Going to like media with hashtag #like4like.
2020-01-16 20:48:22,277 - INFO - Stats saved at 2020-01-16 20:48:22.
2020-01-16 20:48:25,037 - INFO - Going to like 29 medias.
2020-01-16 20:52:35,207 - ERROR - Request returns 400 error!
2020-01-16 20:52:35,209 - INFO - Instagram's error message: Media not found or unavailable
2020-01-16 20:52:35,210 - INFO - Instagram's error message: {}
2020-01-16 21:02:11,528 - ERROR - Request returns 400 error!

**Comment by Abdulsamipy** [comment]
@fighterii i'm not a expert but Instagram has changed its HTML code .. instapy bot was facing like and comment issue too they were able to fix it by changing the Xpath

**Comment by Abdulsamipy** [comment]
so i updated the files  and its still the same not liking photos by hashtags
 **bot** Example/multi_script_CLI.py

Name: instabot
Version: 0.105.0
Summary: Instagram bot scripts for promotion and API python wrapper.
Home-page: https://github.com/instagrambot/instabot
Author: Daniil Okhlopkov, Evgeny Kemerov
Author-email: danokhlopkov@gmail.com, eskemerov@gmail.com
License: Apache Software License 2.0
Location: c:\users\lenovo\appdata\local\programs\python\python38\lib\site-packages
Requires: tqdm, requests, requests-toolbelt, schedule, pysocks, responses, future, six, huepy, pytz
Required-by:

**log:**

________________________________________________________________________________________________
2020-01-20 18:21:28,358 - INFO - Going to like 68 medias.
  0%|                                                                                                                                           | 0/68 [00:00<?, ?it/s]2020-01-20 18:21:28,713 - DEBUG - GET to endpoint: media/2225551819367289294_27780149617/info/ returned response: <Response [200]>
2020-01-20 18:21:29,083 - DEBUG - GET to endpoint: media/2225551819367289294_27780149617/info/ returned response: <Response [200]>
2020-01-20 18:21:29,450 - DEBUG - GET to endpoint: media/2225551819367289294_27780149617/comments/ returned response: <Response [200]>
  1%|█▉                                                                                                                                 | 1/68 [00:11<12:23, 11.10s/it]2020-01-20 18:21:40,162 - DEBUG - GET to endpoint: media/2225551783446733573_11941534388/info/ returned response: <Response [200]>
2020-01-20 18:21:40,776 - DEBUG - GET to endpoint: media/2225551783446733573_11941534388/info/ returned response: <Response [200]>
2020-01-20 18:21:41,287 - DEBUG - GET to endpoint: media/2225551783446733573_11941534388/comments/ returned response: <Response [200]>
  3%|███▊                                                                                                                               | 2/68 [00:22<12:27, 11.32s/it]2020-01-20 18:21:52,142 - DEBUG - GET to endpoint: media/2225551470249841273_22441164564/info/ returned response: <Response [200]>
2020-01-20 18:21:52,856 - DEBUG - GET to endpoint: media/2225551470249841273_22441164564/info/ returned response: <Response [200]>
2020-01-20 18:21:53,627 - DEBUG - GET to endpoint: media/2225551470249841273_22441164564/comments/ returned response: <Response [200]>
  4%|█████▊                                                                                                                             | 3/68 [00:35<12:35, 11.63s/it]2020-01-20 18:22:04,328 - DEBUG - GET to endpoint: media/2225551411521252818_11941534388/info/ returned response: <Response [200]>
2020-01-20 18:22:05,250 - DEBUG - GET to endpoint: media/2225551411521252818_11941534388/info/ returned response: <Response [200]>
2020-01-20 18:22:05,681 - DEBUG - GET to endpoint: media/2225551411521252818_11941534388/comments/ returned response: <Response [200]>

**Comment by jhumms** [comment]
Any update on the fix? @fighterii

**Comment by bruvv** [comment]
@jhumms can you run the development branch? There is a better log options to debug this issue.

**Comment by jhumms** [comment]
Will do, I'll have the results soon.

**Comment by very-sad-cat** [comment]
I think I've found the solution, change this line
`sys.path.append(os.path.join(sys.path[0], "../"))` to  `sys.path.insert(0,os.path.join(sys.path[0], "../"))`. 

I've raised this issue https://github.com/instagrambot/instabot/issues/1240. 

I have the full logs now !

**Comment by Abdulsamipy** [comment]
> I think I've found the solution, change this line
> `sys.path.append(os.path.join(sys.path[0], "../"))` to `sys.path.insert(0,os.path.join(sys.path[0], "../"))`.
> 
> I've raised this issue #1240.
> 
> I have the full logs now !

can you kindly guide where to add the code?

**Comment by Abdulsamipy** [comment]
so i tried to run ultimate.py .. it'snot liking the post either


**here's the log file** 
[instabot_1716914682752.log](https://github.com/instagrambot/instabot/files/4093413/instabot_1716914682752.log)

**Comment by jhumms** [comment]
@Abdulsamipy that was the one I was running. Which, @fighterii  I have the results, but the error log doesn't seem to be much more detailed than the last:
2020-01-21 15:26:52,369 - INFO - Going to like 29 medias.
2020-01-21 15:29:54,409 - ERROR - Request returns 400 error!
2020-01-21 15:29:54,412 - INFO - Instagram's error message: Media not found or unavailable
2020-01-21 15:29:54,414 - INFO - Instagram's error message: {}
2020-01-21 15:37:32,465 - ERROR - Request returns 400 error!
2020-01-21 15:37:32,469 - INFO - Instagram's error message: Media not found or unavailable
2020-01-21 15:37:32,470 - INFO - Instagram's error message: {}
2020-01-21 15:40:55,217 - INFO - DONE: Total liked 0 medias.
2020-01-21 16:26:47,963 - INFO - Going to like media with hashtag #cat.
2020-01-21 16:26:48,611 - INFO - Stats saved at 2020-01-21 16:26:48.
2020-01-21 16:26:53,018 - INFO - Going to like 29 medias.
2020-01-21 16:39:41,527 - INFO - DONE: Total liked 0 medias.

**Comment by Abdulsamipy** [comment]
hi, guys so i was running the script and today i got a new issue hope this will help figure out the problem
**bot**:  Multiscript
**Objective**: Like by hashtags

![Capture](https://user-images.githubusercontent.com/58009877/72913077-6b9ea280-3d5e-11ea-9208-c712f015d7d0.PNG)

**Comment by Abdulsamipy** [comment]
i am not a expert ..but instagram has changed its HTML ..instapy was able to solve like and comment issue by replacing the path

**Comment by jhumms** [comment]
any update?

**Comment by bruvv** [comment]
Still working on it see Dev branch

**Comment by ElonPark** [comment]
I think `check_media()` is the problem.
If set the parameter `check_media = False` in` bot.like ', it will work.

**Comment by Abdulsamipy** [comment]
> check_media

so i edited  `check_media ()` in bot_like and  to "false" in both `def like_medias` and `def like(`  still not liking the media by hashtag

**Comment by ElonPark** [comment]
> I think `check_media ()` is the problem.
> If set the parameter `check_media = False` in` bot.like ', it will work.

I suspect `self.api.media_info(media_id)` in `check_media()`.

**Comment by Abdulsamipy** [comment]
> > I think `check_media ()` is the problem.
> > If set the parameter `check_media = False` in` bot.like ', it will work.
> 
> I suspect `self.api.media_info(media_id)` in `check_media()`.

![Capture](https://user-images.githubusercontent.com/58009877/73171744-51304480-4123-11ea-9573-d3aeb817380d.PNG)
 im in bot_like.py .. is this it??

**Comment by ElonPark** [comment]
I tested like this and it works fine.

```python3
hashtag_medias = self.bot.get_total_hashtag_medias(hashtag,  amount=90, filtration=False)
hashtag_id = self.hashtag_id(hashtag)
feed_position = 0

for media in hashtag_medias:
    try:
        self.likeBy(feed_position, hashtag, hashtag_id, media)
     except Exception as error:
         print(error)
    finally:
                feed_position += 1

def likeBy(self, hashtag, hashtag_id, media):
    like_result = False
    if self.bot.reached_limit("likes"):
        return like_result
        
    like_result = self.bot.like(
        media,
        check_media=False,
        container_module='feed_contextual_hashtag',
        feed_position=feed_position,
        hashtag_name=hashtag,
        hashtag_id=hashtag_id
    )
    print("like action: {}".format(like_result))

    return like_result

def hashtag_id(self, hashtag):
    hashtag_id = None
    if self.bot.api.search_tags(hashtag):
        for tag in self.bot.api.last_json["results"]:
            if tag["name"] == hashtag:
                hashtag_id = tag["id"]
                return hashtag_id
        else:
            self.bot.logger.error(
                "Hashtag ID of {} not found within api response".format(
                    hashtag)
            )
            self.bot.logger.debug(
                "Last JSON results: {}".format(
                    self.bot.api.last_json["results"])
            )
            return hashtag_id
```

**Comment by Abdulsamipy** [comment]
> I tested like this and it works fine.
> 
> ```python
> hashtag_medias = self.bot.get_total_hashtag_medias(hashtag,  amount=90, filtration=False)
> hashtag_id = self.hashtag_id(hashtag)
> feed_position = 0
> 
> for media in hashtag_medias:
>     try:
>         self.likeBy(feed_position, hashtag, hashtag_id, media)
>      except Exception as error:
>          print(error)
>     finally:
>                 feed_position += 1
> 
> def likeBy(self, hashtag, hashtag_id, media):
>     like_result = False
>     if self.bot.reached_limit("likes"):
>         return like_result
>         
>     like_result = self.bot.like(
>         media,
>         check_media=False,
>         container_module='feed_contextual_hashtag',
>         feed_position=feed_position,
>         hashtag_name=hashtag,
>         hashtag_id=hashtag_id
>     )
>     print("like action: {}".format(like_result))
> 
>     return like_result
> 
> def hashtag_id(self, hashtag):
>     hashtag_id = None
>     if self.bot.api.search_tags(hashtag):
>         for tag in self.bot.api.last_json["results"]:
>             if tag["name"] == hashtag:
>                 hashtag_id = tag["id"]
>                 return hashtag_id
>         else:
>             self.bot.logger.error(
>                 "Hashtag ID of {} not found within api response".format(
>                     hashtag)
>             )
>             self.bot.logger.debug(
>                 "Last JSON results: {}".format(
>                     self.bot.api.last_json["results"])
>             )
>             return hashtag_id
> ```

pardon me ... can you tell me from which file you copied this set of code. .. cause i can't seem to find the code inside bot_like.py

you can check here:
https://github.com/instagrambot/instabot/blob/master/instabot/bot/bot_like.py

**Comment by ElonPark** [comment]
> > I tested like this and it works fine.
> > ```python
> > hashtag_medias = self.bot.get_total_hashtag_medias(hashtag,  amount=90, filtration=False)
> > hashtag_id = self.hashtag_id(hashtag)
> > feed_position = 0
> > 
> > for media in hashtag_medias:
> >     try:
> >         self.likeBy(feed_position, hashtag, hashtag_id, media)
> >      except Exception as error:
> >          print(error)
> >     finally:
> >                 feed_position += 1
> > 
> > def likeBy(self, hashtag, hashtag_id, media):
> >     like_result = False
> >     if self.bot.reached_limit("likes"):
> >         return like_result
> >         
> >     like_result = self.bot.like(
> >         media,
> >         check_media=False,
> >         container_module='feed_contextual_hashtag',
> >         feed_position=feed_position,
> >         hashtag_name=hashtag,
> >         hashtag_id=hashtag_id
> >     )
> >     print("like action: {}".format(like_result))
> > 
> >     return like_result
> > 
> > def hashtag_id(self, hashtag):
> >     hashtag_id = None
> >     if self.bot.api.search_tags(hashtag):
> >         for tag in self.bot.api.last_json["results"]:
> >             if tag["name"] == hashtag:
> >                 hashtag_id = tag["id"]
> >                 return hashtag_id
> >         else:
> >             self.bot.logger.error(
> >                 "Hashtag ID of {} not found within api response".format(
> >                     hashtag)
> >             )
> >             self.bot.logger.debug(
> >                 "Last JSON results: {}".format(
> >                     self.bot.api.last_json["results"])
> >             )
> >             return hashtag_id
> > ```
> 
> pardon me ... can you tell me from which file you copied this set of code. .. cause i can't seem to find the code inside bot_like.py
> 
> you can check here:
> https://github.com/instagrambot/instabot/blob/master/instabot/bot/bot_like.py

oh! sorry this code is in my custom class

**Comment by Abdulsamipy** [comment]
> 
> oh! sorry this code is in my custom class

oh .. is it possible for you to update this custom class to dev branch?

**Comment by ElonPark** [comment]
> > oh! sorry this code is in my custom class
> 
> oh .. is it possible for you to update this custom class to dev branch?

I am writing test code for you right now.

**Comment by ElonPark** [comment]
> > oh! sorry this code is in my custom class
> 
> oh .. is it possible for you to update this custom class to dev branch?

@Abdulsamipy please try this
https://gist.github.com/ElonPark/43df4fe7df78b0cc013b654cefd2a60c

**Comment by Abdulsamipy** [comment]
`2020-01-27 17:29:48,521 - DEBUG - GET to endpoint: tags/search/?is_typeahead=true&q=cat&rank_token=27194590479_6cfe777e-001d-4d67-aa62-352e1e261efb returned response: <Response [200]>
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given
likeBy() takes 4 positional arguments but 5 were given

likeBy() takes 4 positional arguments but 5 were given
2020-01-27 17:29:48,687 - INFO - Total requests: 31`

**Comment by ElonPark** [comment]
>`2020-01-27 17:29:48,521 - DEBUG - GET to endpoint: tags/search/?is_typeahead=true&q=cat&rank_token=27194590479_6cfe777e-001d-4d67-aa62-352e1e261efb returned response: <Response [200]>
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>likeBy() takes 4 positional arguments but 5 were given
>
>likeBy() takes 4 positional arguments but 5 were given
>2020-01-27 17:29:48,687 - INFO - Total requests: 31`

sorry my mistake
i fix it
https://gist.github.com/ElonPark/43df4fe7df78b0cc013b654cefd2a60c

**Comment by Abdulsamipy** [comment]
> > `2020-01-27 17 : 29 : 48,521-DEBUG-엔드 포인트로 가져 오기 : tags / search /? is_typeahead = true & q = cat & rank_token = 27194590479_6cfe777e-001d-4d67-aa62-352e1e261efb 반환 응답 : <Response [200]>
> > likeBy () 4 개의 위치 인수를 취하지 만 5는
> > likeBy ()가 주어짐 5 는 4 개의 위치 인수를 갖지만 5는
> > likeBy ()가 4 개의 위치 인수를 취했지만 5는
> > likeBy ()가 4 개의 위치 인수를 취하지 만 5는
> > likeBy ()가 4의 위치 인수를 취하지 만 5 주어진
> > likeBy ()을 4 개 위치 인수하지만 소요 5 주어진
> > likeBy ()을 4 개 위치 인수 걸리지 만 5 주어진
> > likeBy ()는 4 개의 위치 인수를 취하지 만 5는
> > 2020-01-27 17 : 29 : 48,687을 받았습니다-정보-총 요청 : 31`
> 
> sorry my mistake
> i fix it
> https://gist.github.com/ElonPark/43df4fe7df78b0cc013b654cefd2a60c

GREAT!! ITS WORKING ..  can you kindly tell me how can i add this to multi_script_CLI.py

https://github.com/instagrambot/instabot/blob/master/examples/multi_script_CLI.py

**Comment by ElonPark** [comment]
> > > `2020-01-27 17 : 29 : 48,521-DEBUG-엔드 포인트로 가져 오기 : tags / search /? is_typeahead = true & q = cat & rank_token = 27194590479_6cfe777e-001d-4d67-aa62-352e1e261efb 반환 응답 : <Response [200]> likeBy () 4 개의 위치 인수를 취하지 만 5는 likeBy ()가 주어짐 5 는 4 개의 위치 인수를 갖지만 5는 likeBy ()가 4 개의 위치 인수를 취했지만 5는 likeBy ()가 4 개의 위치 인수를 취하지 만 5는 likeBy ()가 4의 위치 인수를 취하지 만 5 주어진 likeBy ()을 4 개 위치 인수하지만 소요 5 주어진 likeBy ()을 4 개 위치 인수 걸리지 만 5 주어진 likeBy ()는 4 개의 위치 인수를 취하지 만 5는 2020-01-27 17 : 29 : 48,687을 받았습니다-정보-총 요청 : 31`
> > 
> > 
> > sorry my mistake
> > i fix it
> > https://gist.github.com/ElonPark/43df4fe7df78b0cc013b654cefd2a60c
> 
> GREAT!! ITS WORKING .. can you kindly tell me how can i add this to multi_script_CLI.py
> 
> https://github.com/instagrambot/instabot/blob/master/examples/multi_script_CLI.py

This is a temporary measure to avoid the problem, not the right solution.

**Comment by Abdulsamipy** [comment]
> > > > `2020-01-27 17 : 29 : 48,521-DEBUG-엔드 포인트로 가져 오기 : tags / search /? is_typeahead = true & q = cat & rank_token = 27194590479_6cfe777e-001d-4d67-aa62-352e1e261efb 반환 응답 : <Response [200]> likeBy () 4 개의 위치 인수를 취하지 만 5는 likeBy ()가 주어짐 5 는 4 개의 위치 인수를 갖지만 5는 likeBy ()가 4 개의 위치 인수를 취했지만 5는 likeBy ()가 4 개의 위치 인수를 취하지 만 5는 likeBy ()가 4의 위치 인수를 취하지 만 5 주어진 likeBy ()을 4 개 위치 인수하지만 소요 5 주어진 likeBy ()을 4 개 위치 인수 걸리지 만 5 주어진 likeBy ()는 4 개의 위치 인수를 취하지 만 5는 2020-01-27 17 : 29 : 48,687을 받았습니다-정보-총 요청 : 31`
> > > 
> > > 
> > > sorry my mistake
> > > i fix it
> > > https://gist.github.com/ElonPark/43df4fe7df78b0cc013b654cefd2a60c
> > 
> > 
> > GREAT!! ITS WORKING .. can you kindly tell me how can i add this to multi_script_CLI.py
> > https://github.com/instagrambot/instabot/blob/master/examples/multi_script_CLI.py
> 
> This is a temporary measure to avoid the problem, not the right solution.

understood .. thanks alot

**Comment by bruvv** [comment]
I update the master branch. Please retest

**Comment by Abdulsamipy** [comment]
> I update the master branch. Please retest

so i updated the bot and installed the master branch, tested following bots 

**Ultimate.py** : Not liking the hashtags "getting same log" 
_____________________________________________________________________________________________
` 9%|███████████                                                                                                                      | 7/82 [06:19<1:11:25, 57.14s/it]2020-01-28 15:16:16,507 - DEBUG - GET to endpoint: media/2231253368547012614_7384072680/info/ returned response: <Response [200]>
2020-01-28 15:16:17,120 - DEBUG - GET to endpoint: media/2231253368547012614_7384072680/info/ returned response: <Response [200]>
2020-01-28 15:16:17,682 - DEBUG - GET to endpoint: media/2231253368547012614_7384072680/comments/ returned response: <Response [200]>`
_____________________________________________________________________________________________
**Multi_script_Cli.py**: not starting .. "name 'settings' is not defined"
"log" 
_____________________________________________________________________________________________
`, in setting_input
    with open(settings, "w") as f:
NameError: name 'settings' is not defined`
_____________________________________________________________________________________________

`

**Comment by Abdulsamipy** [comment]
Hey guys, any update?

**Comment by bruvv** [comment]
Yes in the development branch...

**Comment by Abdulsamipy** [comment]
@bruvv  thanks for your effort .. but Its still the same .. not liking by hashtags

`what hashtags?
cat
2020-02-02 16:21:11,263 - INFO - Going to like media with hashtag #cat.
Getting hashtag media.: 0it [00:00, ?it/s]2020-02-02 16:21:14,960 - DEBUG - GET to endpoint: feed/tag/cat/?max_id=&rank_token=27194590479_721d4d07-ab25-4b06-924e-726adcef1dfe&ranked_content=true& returned response: <Response [200]>
2020-02-02 16:21:15,568 - DEBUG - GET to endpoint: tags/search/?is_typeahead=true&q=cat&rank_token=27194590479_721d4d07-ab25-4b06-924e-726adcef1dfe returned response: <Response [200]>
2020-02-02 16:21:15,571 - INFO - Going to like 86 medias.
  0%|                                                                                           | 0/86 [00:00<?, ?it/s]2020-02-02 16:21:16,181 - DEBUG - GET to endpoint: media/2234913471189759863_29323074844/info/ returned response: <Response [200]>
2020-02-02 16:21:16,589 - DEBUG - GET to endpoint: media/2234913471189759863_29323074844/info/ returned response: <Response [200]>
2020-02-02 16:21:17,002 - DEBUG - GET to endpoint: media/2234913471189759863_29323074844/comments/ returned response: <Response [200]>
  1%|▉                                                                                  | 1/86 [00:11<16:12, 11.44s/it]2020-02-02 16:21:27,447 - DEBUG - GET to endpoint: media/2234913442509430961_28004739581/info/ returned response: <Response [200]>
2020-02-02 16:21:27,852 - DEBUG - GET to endpoint: media/2234913442509430961_28004739581/info/ returned response: <Response [200]>
2020-02-02 16:21:28,302 - DEBUG - GET to endpoint: media/2234913442509430961_28004739581/comments/ returned response: <Response [200]>
  2%|█▉                                                                                 | 2/86 [00:22<15:56, 11.39s/it]2020-02-02 16:22:26,634 - DEBUG - GET to endpoint: media/2234913436840325087_2573140/info/ returned response: <Response [200]>
2020-02-02 16:22:27,552 - DEBUG - GET to endpoint: media/2234913436840325087_2573140/info/ returned response: <Response [200]>
2020-02-02 16:22:28,039 - DEBUG - GET to endpoint: media/2234913436840325087_2573140/comments/ returned response: <Response [200]>`

**Comment by jkw935** [comment]
+1 same here

**Comment by ching222** [comment]
I mostly only use the hashtags and users followers functions and I too is getting the same response 200, from both of these functions. I see that lots of the others are having the same issue too, is there any info of the error that I should provide?

**Comment by ching222** [comment]
anyone still having the response 200 problem..?

**Comment by matthieuoliveira** [comment]
@ching222  Yes, I pulled most recent version an hour ago '0.107.0'.

Liking individual medias/bulk medias, downloading photos responds with 200 but does actually make the request. With that said, get_total_user_medias worked fine, get last user medias, get follower info, ids is fine. Looks like this version updated yesterday so need to dig into commits to see what actually changed.

edit: 

setting  check_media = False 
resolved it for like() - will need to try other functions

**Comment by bruvv** [comment]
I am not following you @matthieuoliveira what is the error to begin with and what fixed  setting check_media ?

**Comment by matthieuoliveira** [comment]
@bruvv 
Sure,

` bot.like(media_id=media_id, username=account, check_media = False)` given check_media is False,  it returns 200  and the media is liked, given it's true, it never likes the media. Sorry if I wasn't clear. I can test out all endpoints on my end if that will be helpful?

For example, 

script.py
```
bot = Bot(base_path=paths['log_path'] , save_logfile=False)
bot.login(username=login['username'], password=login['password'])
bot.like(media_id, user_id=user_id, check_media=True)
```

logs
```
2020-02-14 06:09:47,865 - INFO - Instabot version: 0.107.0 Started
2020-02-14 06:09:47,865 - DEBUG - Bot imported from /usr/local/lib/python3.8/site-packages/instabot/bot/bot.py
2020-02-14 06:09:47,868 - INFO - PRE-LOGIN FLOW!...
2020-02-14 06:09:50,075 - DEBUG - POST to endpoint: accounts/read_msisdn_header/ returned response: <Response [200]>
2020-02-14 06:09:51,235 - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-02-14 06:09:53,356 - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-14 06:09:55,488 - DEBUG - POST to endpoint: attribution/log_attribution/ returned response: <Response [200]>
2020-02-14 06:09:57,621 - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-02-14 06:10:00,118 - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [200]>
2020-02-14 06:10:00,119 - INFO - Logged-in successfully as 'myaccount123'!
2020-02-14 06:10:00,119 - INFO - LOGIN FLOW! Just logged-in: True
2020-02-14 06:10:01,344 - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-02-14 06:10:02,867 - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-14 06:10:06,500 - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-02-14 06:10:08,911 - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-02-14 06:10:11,139 - DEBUG - POST to endpoint: fbsearch/suggested_searches/ returned response: <Response [200]>
2020-02-14 06:10:13,393 - DEBUG - POST to endpoint: fbsearch/suggested_searches/ returned response: <Response [200]>
2020-02-14 06:10:15,645 - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [200]>
2020-02-14 06:10:17,897 - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [200]>
2020-02-14 06:10:20,679 - DEBUG - POST to endpoint: direct_v2/inbox/ returned response: <Response [200]>
2020-02-14 06:10:21,996 - DEBUG - GET to endpoint: direct_v2/get_presence/ returned response: <Response [200]>
2020-02-14 06:10:24,578 - DEBUG - GET to endpoint: news/inbox/?limited_activity=true&show_su=true returned response: <Response [200]>
2020-02-14 06:10:26,886 - DEBUG - GET to endpoint: loom/fetch_config/ returned response: <Response [200]>
2020-02-14 06:10:28,119 - DEBUG - GET to endpoint: users/profile_notice/ returned response: <Response [200]>
2020-02-14 06:10:29,522 - DEBUG - POST to endpoint: qp/batch_fetch/ returned response: <Response [200]>
2020-02-14 06:10:31,794 - DEBUG - POST to endpoint: discover/explore/ returned response: <Response [200]>
2020-02-14 06:10:31,800 - DEBUG - Loading Checkpoint file from: /app/config/myaccount123.checkpoint
2020-02-14 06:10:34,137 - DEBUG - GET to endpoint: users/target_account_123/usernameinfo/ returned response: <Response [200]>
2020-02-14 06:10:36,900 - DEBUG - GET to endpoint: media/media_id_123/info/ returned response: <Response [200]>
2020-02-14 06:10:39,207 - DEBUG - GET to endpoint: media/media_id_123/info/ returned response: <Response [200]>
2020-02-14 06:10:40,472 - DEBUG - GET to endpoint: media/media_id_123/comments/ returned response: <Response [200]>
2020-02-14 06:10:40,474 - DEBUG - Saving Checkpoint file to: /app/config/myaccount123.checkpoint
2020-02-14 06:10:40,477 - INFO - Total requests: 25
```

**Comment by andreagiugio** [comment]
I got the same error. Script is not able to like medias

**Comment by Abdulsamipy** [comment]
> bot.like(media_id=media_id, username=account, check_media = False)

hey, can you tell me where do i have to add this code 
`bot.like(media_id=media_id, username=account, check_media = False)`  in multi_script_CLI
